deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -114,6 +115,7 @@ def _execute_compaction(
     compaction_audit = CompactionSessionAuditInfo(deltacat.__version__, audit_url)
 
     compaction_audit.set_hash_bucket_count(params.hash_bucket_count)
```

Comment:
Nit: Could we inline this as `compaction_audit.set_hash_bucket_count(params.hash_bucket_count).set_compactor_version(CompactorVersion.V2.value)` since we are using a builder pattern for the `CompactionSessionAuditInfo` object

Reply:
Sure.